### PR TITLE
Changed how .cmfconfig file was saved, retrieved and displayed and ad…

### PR DESCRIPTION
**Pull request by varkha-d-sharma**
…ded neo4j options in cmf init.

### CODE REVIEW

File: cmflib/cmf.py [hunk 1]
```diff
@@ -56,20 +57,22 @@ class Cmf:
...
             ```
     """
 
     # pylint: disable=too-many-instance-attributes
-
-    __neo4j_uri = os.getenv('NEO4J_URI', "")
-    __neo4j_user = os.getenv('NEO4J_USER_NAME', "")
-    __neo4j_password = os.getenv('NEO4J_PASSWD', "")
+    # Reading CONFIG_FILE variable
+    cmf_config = os.environ.get("CONFIG_FILE",".cmfconfig")
```
Comment (annmary-roy) [inline cmflib/cmf.py:70]: Where are we setting this environment variable ?

Comment (annmary-roy) [reply cmflib/cmf.py:70]: Add documentation for CONFIG_FILE env variable

File: cmflib/commands/init/amazonS3.py [hunk 2]
```diff
@@ -21,22 +21,51 @@
...
+            CmfConfig.write_config(cmf_config, "cmf", attr_dict, True)
+
+        # read --neo4j details and add to the exsting file
+        if self.args.neo4j_user and self.args.neo4j_password and self.args.neo4j_uri:
+            attr_dict = {}
+            attr_dict["user"] = self.args.neo4j_user
+            attr_dict["password"] = self.args.neo4j_password
+            attr_dict["uri"] = self.args.neo4j_uri
+            CmfConfig.write_config(cmf_config, "neo4j", attr_dict, True)
+        elif (
```
[the remark this answers is not in the document]
Comment (varkha-d-sharma) [reply cmflib/commands/init/amazonS3.py:61]: it is needed in case user doesn't set neo4j at all

File: cmflib/cmf.py (hunk 1 above)
Comment (varkha-d-sharma) [reply cmflib/cmf.py:70]: Will do.

**Review by annmary-roy — APPROVED** [review]

**Pull request merged**